src/pages/BuyDomains/BuyDomains.module.scss

```diff
@@ -0,0 +1,68 @@
+@use 'zero-ui/src/styles/dimensions';
+@use 'zero-ui/src/styles/colors';
+@use 'zero-ui/src/styles/typography';
+
+.BuyDomains {
+	&__Page_Container {
+		display: flex;
+		justify-content: center;
+		align-items: center;
+		min-height: calc(100vh - 150px);
+	}
+
+	&__Container {
+		display: flex;
+		flex-direction: column;
+		justify-content: center;
+		align-items: center;
+		min-width: 550px;
+		max-width: 100%;
+	}
+
+	&__Header {
+		display: flex;
+		align-content: center;
+		gap: 24px;
+
+		h1 {
+			color: colors.$text-primary;
+			font-size: typography.$size-xlarge;
+			font-weight: typography.$font-weight-regular;
+			line-height: 39px;
+			letter-spacing: 0.2em;
```

Comment:
I'm seeing a mixture of `px`, `em`, and `rem` in this file. Is there a reason for this that I'm missing or was this just habit from using `px` in `zNS-dapp`?